src/ui/include/EditorNS/editor.h

```diff
@@ -265,17 +243,18 @@ namespace EditorNS
         QList<Theme> themes();
         void setTheme(Theme theme);
         static Editor::Theme themeFromName(QString name);
-
-        QList<Selection> selections();
-
+ 
         /**
          * @brief Returns the currently selected texts.
          * @return
          */
-        Q_INVOKABLE QStringList selectedTexts();
+        Q_INVOKABLE void setSelectionsText(const QStringList &texts, selectMode mode);
```

Comment:
While we're on the subject... would you be against prepending "get"  to these functions so the API is a little more consistent?  Right now its kind of back and forth depending on which part of Notepadqq's source you're looking at... so I'd like to have a standard for this.